packages/vertexai/lib/constants.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { version } from './version';
+
+export const VERTEX_TYPE = 'vertexAI';
+
+export const DEFAULT_LOCATION = 'us-central1';
+
+export const DEFAULT_BASE_URL = 'https://firebasevertexai.googleapis.com';
+
+export const DEFAULT_API_VERSION = 'v1beta';
```

Comment:
This seems like it might need maintenance over time - a comment explaining (ideally with a link ?) the source of these would help future-you / future-us